.gitignore

```diff
@@ -53,3 +53,9 @@ weights
 *.lprof
 *.pkl
 site/
+.clang-format
+.tool-versions
+setup_extra.py
```

Comment:
unrelated change, why it's here?

Reply:
it's just some local files I don't want to push, if not in .gitignore , how should i make git ignore it

Reply:
This is not resolved

Reply:
ok. fixed